tests/ha/fencing.pm

```diff
@@ -36,6 +45,8 @@ sub run {
         assert_script_run 'crm -F node fence ' . get_node_to_join if is_node(2);
     }
 
+    sleep bmwqemu::scale_timeout(300) if (check_var('AUTOMATED_REGISTER', 'false') && get_var('HA_CLUSTER_INIT'));
```

Comment:
Couple of questions:

1. Any reason this is before the block between L50 and L59? I think it should go after to ensure the sleep is called after the node starts to fence, but I don't know if you've found a race condition if done that way.
2. Why not add `is_sles4sap` to the conditional? I get that it's only going to be called when `AUTOMATED_REGISTER == false` and currently we're only using that variable in the hanaSR tests, but IMO the variable name sounds too general, and can lead to confusion.